### ISSUE TITLE
Trying with MobileNet-V2

### ISSUE THREAD

**Issue by anerisheth19** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#48]
I am trying to replace Mobilenet-V1 with Mobilenet-V2. Any insights?

**Comment by Daniil-Osokin** [comment]
Please check the paper. In short, it did not give any speed/accuracy improvement.

**Comment by anerisheth19** [comment]
I did check it. However, I was trying to implement MobileNetV2 on your algorithm. I am getting errors. Can you share the code if you can please?
Thanks,

**Comment by anerisheth19** [comment]
Or if you can share the pretrained model for MobileNet V2, that would be great too!

**Comment by Daniil-Osokin** [comment]
I don't have this code. You may post the error, may be it is possible to debug it.

**Comment by anerisheth19** [comment]
Okay. Thank you!

I read your paper on this implementation and have one question. What does 'cut to conv4_1', 'cut to conv5_5', etc. mean in the MobileNet backbone selection?

**Comment by Daniil-Osokin** [comment]
This means how many layers from backbone take. MobileNet v1 has 6 blocks, which consist of depthwise convolutions and 1x1 pointwise convolutions. After each block stride is grow (there is convolution with stride == 2). First number conv**5**\_5 is block index, the second conv5\_**5**  is index of convolution (depthwise + pointwise) within block. We remove the stride in conv4\_2 and add dilation in conv5\_1, you can check it in the [code](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/models/with_mobilenet.py#L93).

**Comment by anerisheth19** [comment]
Cool. This explains a lot. 

Thanks once again!

**Comment by Daniil-Osokin** [comment]
You are welcome!

**Comment by anerisheth19** [comment]
Is there any specific way you calculated GLOPS and Number of parameters? I am planning to use [https://github.com/sovrasov/flops-counter.pytorch](url)

**Comment by Daniil-Osokin** [comment]
This one is ok, looks like we have used it. You can also install it through [pip](https://pypi.org/project/ptflops/).
